Master/dependency-reduced-pom.xml

```diff
@@ -1,44 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
I plan to fix the issue that this file gets a reformat every single pull request. Maybe i should remove these poms from the entire project. To be honest, i don't even know why they are here in first place....